src/shared/validate/validate.ts

```diff
@@ -3,14 +3,17 @@ import { getLogger } from "../logger";
 import { SchemaId } from "../@types/document";
 // don't change this otherwise there is a cycle
 import { getData } from "../utils/utils";
+import { Kind } from "../utils/diagnose";
```

Comment:
this creates a direct cycle. Move it our to a shared file

Reply:
done.